daq/panda/test/interpreter.py

```diff
@@ -0,0 +1,108 @@
+import os.path 
+import serial
+print(serial.__file__),
+import pyrebase
+from time import time, sleep
+from datetime import datetime
+
+config = { #firebase authentification
+    "apiKey": "AIzaSyDEfkwr7Zl5WucXFluMxB8VIlngUnp7aDM",
+    "authDomain": "smv-daq.firebaseapp.com",
+    "databaseURL": "https://smv-daq.firebaseio.com",
+    "projectId": "smv-daq",
+    "storageBucket": "bucket.appspot.com"
+}
+
+firebase = pyrebase.initialize_app(config)
+
+db = firebase.database()
+
+#declare the dictionary to store all input data
+inputs_dict={
+    'Cur' : -1,'Vlt' : -1,'Thr' : -1,'Pwr' : -1,
+    'Spd' : -1,'Lng' : -1,'Lat' : -1,'Alt' : -1,'Tem' : -1,
+    'GyX' : -1,'GyY' : -1,'GyZ' : -1,'AcX' : -1,'AcY' : -1,'AcZ' : -1,'MaX' : -1,'MaY' : -1,'MaZ' : -1,
+    'Pit' : -1,'Rol' : -1,'Hea' : -1,'Rpm' : -1, 
+    }
+
+prefix = ['Cur', 'Vlt', 'Thr', 'Pwr', 'Spd', 'Lng', 'Lat', 'Alt', 'Tem', 
+             'GyX', 'GyY', 'GyZ', 'AcX', 'AcY', 'AcZ', 'MaX', 'MaY', 'MaZ', 
+             'Pit', 'Rol', 'Hea', 'Rpm']
+
+Trial_Number = input("Enter Trial Number \n") 
+
+#Reads in black box data to certain location
+save_path = 'C:\\Users\\jsand\\OneDrive\\Documents\\SMV\\DAQ\\Trials' #sets file location for where the text files will be read from
+trialName = "Trial " + Trial_Number
+fileName =  trialName + ".txt"
+completeName = os.path.join(save_path, fileName)
+
+#Printing inputs to Black Box
+text = open(completeName, 'r')
+
+for position, line in enumerate(text):
+    if position%3 == 0:  #reads every third line  
+        
+    #Setting the time for each trial
+        #now = datetime.now() #set time
+        
+
+        numValues = 22 #expected number of data inputs
+        data = line.split(".") #splits each data point into a list 
+        timeName = data[22][: -1]
+
+        for i in range (0, numValues):
+                inputs_dict[prefix[i]] = float(data[i]) #may fail on first run, simply re-run
```

Comment:
i feel like we might want to leave in the labels, again in the case that a sensor fails and isn't able to provide its data, so that we don't get misaligned data